assets/theme-css/keyfeatures.css

```diff
@@ -14,12 +14,13 @@
 }
 
 .keyfeatures-box-content {
-  height: 200px;
   min-width: 275px;
   width: 335px;
   margin: 25px auto;
-  border-radius: 25px;
-  border: 4px solid var(--colorBackgroundInverse);
+  border: 0.02em solid var(--pst-color-border);
+  box-shadow:
+    0 0.2rem 0.5rem var(--pst-color-shadow),
+    0 0 0.0625rem var(--pst-color-shadow) !important;
```

Comment:
Why is this !important necessary?

Reply:
I just copied it from pst. I will investigate whether we need it.

Reply:
I removed it.